be1-go/hub/inbox.go

```diff
@@ -75,3 +114,73 @@ func (i *inbox) getMessage(messageID []byte) (*message.Message, bool) {
 	}
 	return msgInfo.message, true
 }
+
+func (i *inbox) storeMessageInDB(messageInfo *messageInfo) error {
+	db, err := sql.Open("sqlite3", os.Getenv("HUB_DB"))
```

Comment:
This will eventually be abstracted away but it would be nice if we can have a long living *sql.DB instance and pass that everywhere for the life of the hub.

Reply:
Conclusion from slack: to be taken up separately when dealing with Storage abstraction